rpc/cloudagent/security/role/moc_cloudagent_role.proto

```diff
@@ -28,9 +29,12 @@ message RoleResponse {
 }
 
 message Action {
-	AccessOperation operation = 1;
+	// General access permissions
+	GeneralAccessOperation operation = 1;
```

Comment:
since we are reusing the operation variable, and changing its type we might run into backwards compatibility issues with grpc. Might need to create a new var GeneralAccessOperation operation = 4; and mark old as deprecated